README.md

```diff
@@ -1,21 +1,25 @@
-This is a small windows script to help extract Wings! amiga disks from the GOG release.
+This is a small Windows script to help extract Wings! amiga disks and the Kickstart 1.2 firmware from the GOG release. Also possible to extract from Linux, as explained below.
```

Comment:
`extract from Linux`

Please change the grammar here, to something other than from Linux. Maybe using linux or running linux, or reword the whole sentence! :+1:

Reply:
I'll see how I can reword it. Thanks for pointing out.